packages/Utils/src/DTK_ConfigDefs.hpp

```diff
@@ -26,7 +29,11 @@ namespace DataTransferKit
 #include "DTK_Types.h"
 
 // clang-format off
+#ifdef HAVE_DTK_BOOST
 #define REGION_NAME(x) BOOST_CURRENT_FUNCTION+std::string(":")+std::string(x)
+#else
+#define REGION_NAME(x) std::string(x)
```

Comment:
I would prefix `REGION_NAME` with `DTK::` when compiling without boost so that in a larger application you could still understand that the parallel region corresponds to DTK.

Reply:
This is something I want to change as well.
It does not long belong to this PR though.  I will open an issue to track it.

Reply:
For reference, I opened #347